nengo/_vendor/npconv2d/conv2d.py

```diff
@@ -185,21 +185,30 @@ def conv2d(x, w, pad='SAME', stride=(1, 1)):
     """2D convolution (technically speaking, correlation).
     Args:
         x: [N, H, W, C]
-        w: [I, J, C, K]
+        w: [I, J, C/G, K]
         pad: [PH, PW]
         stride: [SH, SW]
     Returns:
         y: [N, H', W', K]
     """
     ksize = w.shape[:2]
+    c = x.shape[-1]
+    groups = c // w.shape[2] # number of groups inferred
```

Comment:
It might be good to assert that `c % w.shape[2] == 0`. If that isn't the case, my guess is that at best, the output will not be what the user expects (as they probably made a mistake in specifying the weight shape).